recipes/viridis/meta.yaml

```diff
@@ -0,0 +1,67 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "viridis" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "59c0598e0e144bda22fefd0174ad827d095f0ef9baf62f51334436e93b7e748b" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
```

Comment:
Indeed. We are so accustomed to `conda` and `conda-build` that we forgot how weird that can be. Conda-forge has some instructions in the [readme](https://github.com/conda-forge/staged-recipes/#3-how-do-i-exclude-a-platform).

But we could add a link there to the conda-build docs `skip` and `preprocessing-selectors` entries to make it clearer:

http://conda.pydata.org/docs/building/meta-yaml.html#skipping-builds

http://conda.pydata.org/docs/building/meta-yaml.html#preprocessing-selectors